staff/jordi-pujol/cv/index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Jordi_cv</title>
+</head>
+<body>
+    <header>
+        <h1>Jordi Pujol</h1>
+        <img src="https://img.elcomercio.pe/files/article_content_ec_fotos/uploads/2018/10/06/5bb8ce60353c5.jpeg" alt="">
+        <ul>
+            <li><h2>Web Designer Director</h2></li>
```

Comment:
`h2` should not be used to emphasize information. just for marking `headings`. please, remove h2 here and use other convenient tag instead.